### PR TITLE
Rustdoc cleanup

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of https://github.com/rust-lang/rust/issues/83332. The goal of this PR is to remove a few unused things:

 * The "loading content" things are now unneeded.
 * Some toggle CSS rules were still there.
 * Some parts of the JS had a different indent, fixed it.

r? @jsha

### CODE REVIEW

File: src/librustdoc/html/render/print_item.rs [hunk 1]
```diff
@@ -600,7 +596,6 @@ fn item_trait(w: &mut Buffer, cx: &Context<'_>, it: &clean::Item, t: &clean::Tra
         for t in types {
             trait_item(w, cx, t, it);
         }
-        write_loading_content(w, "</div>");
```
Comment (notriddle) [inline src/librustdoc/html/render/print_item.rs:603]: The `</div>` itself still need to be there, doesn't it?

**Review by jsha — COMMENTED** [review]
Generally looks good to me, though I concur with @notriddle's observation that we still need those `</divs>`.

File: src/librustdoc/html/render/print_item.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply src/librustdoc/html/render/print_item.rs:603]: Indeed, good catch!

**Comment by GuillaumeGomez** [comment]
Fixed @notriddle's comment!

**Pull request merged**